### PR TITLE
dynamic gas estimates for splitter txn

### PR DESCRIPTION
**Pull request by owocki**
##### Description

dynamic gas estimates for splitter txn

##### Refers/Fixes

https://twitter.com/lemiscate/status/1242421192889417729?s=12

##### Testing

tested locally

### CODE REVIEW

**Comment by thelostone-mc** [comment]
@owocki looks we've got conflicts 🤔 
Otherwise LGTM

**Comment by owocki** [comment]
merge conflicts fixed

File: app/grants/views.py [hunk 1]
```diff
@@ -1022,7 +1022,6 @@ def grant_fund(request, grant_id, grant_slug):
         'grant_has_no_token': True if grant.token_address == '0x0000000000000000000000000000000000000000' else False,
         'grant': grant,
         'img': img,
-        'clr_prediction_curve': [c[1] for c in grant.clr_prediction_curve] if grant.clr_prediction_curve and len(grant.clr_prediction_curve[0]) > 1 else [0, 0, 0, 0, 0, 0],
```
Comment (danlipert) [inline app/grants/views.py:1025]: is this unused now?

Comment (thelostone-mc) [reply app/grants/views.py:1025]: nope nope we still use this !

**Comment by thelostone-mc** [comment]
@owocki 
- me & dan curious if we are removing code that we need 
- looks like travis is failing.

If those two are resolved we can get this inc

**Comment by owocki** [comment]
just fixed both issues and pushed up. hopefully the travis gods will smileth upon me

**Comment by owocki** [comment]
i just commited the fund.js changes directly to master on 21a3bece8af6c0d5daef22c9b5dbec1c889464fd

**Pull request closed without merging**